AK/Function.h

```diff
@@ -56,6 +56,14 @@ class Function<Out(In...)> {
     using FunctionType = Out(In...);
     using ReturnType = Out;
 
+#ifdef KERNEL
+    constexpr static auto AccommodateExcessiveAlignmentRequirements = false;
+    constexpr static size_t ExcessiveAlignmentThreshold = alignof(void*);
```

Comment:
```suggestion
    constexpr static size_t ExcessiveAlignmentThreshold = alignof(m_storage);
```

Does this not work? It would let us avoid hard-coding it in two different ways based on `KERNEL`.

Reply:
It wouldn't remove the ifdef (the userland case is 16, not `alignof(void*)`), but I'll replace the void* anyway.

Reply:
Huh, I guess not.
```
/home/test/Workspace/serenity/./AK/Function.h:61:67: error: 'm_storage' was not declared in this scope
   61 |     constexpr static size_t ExcessiveAlignmentThreshold = alignof(m_storage);
      |                                                                   ^~~~~~~~~
```